universal/ubi8/Dockerfile

```diff
@@ -440,6 +434,20 @@ cd -
 rm -rf "${TEMP_DIR}"
 EOF
 
+# Bash completions
+RUN dnf -y install bash-completion
+RUN echo "source /etc/profile.d/bash_completion.sh" >> /home/user/.bashrc
+
+## kubectl completion
+RUN kubectl completion bash > /usr/share/bash-completion/completions/kubectl
+
+## nvm completion
+RUN cat ${NVM_DIR}/bash_completion > /usr/share/bash-completion/completions/nvm
+
+# Add sdkman's init script launcher to the end of the .bashrc since we are not adding it on sdkman install
+RUN echo 'export SDKMAN_DIR="/home/user/.sdkman"' >> /home/user/.bashrc
+RUN echo '[[ -s "/home/user/.sdkman/bin/sdkman-init.sh" ]] && source "/home/user/.sdkman/bin/sdkman-init.sh"' >> /home/user/.bashrc
+
```

Comment:
> if we take over setting up `.bashrc` then if sdkman changes something our dockerfile breaks.

I was hesitant to make this change for the same reason. 

I considered curl'ing the sdkman install script and parsing or evaluating the $sdkman_init_snippet which contains the modifications to apply to the `.bashrc` but it seemed like over-kill. Gitpod is doing something [similar](https://github.com/gitpod-io/workspace-images/blob/6e94c2184c21cd3aa1f1343d45253850eecaff32/chunks/lang-java/Dockerfile#L24-L26), but to modify the .bash_profile so that sdkman works in non-interactive shells.